packages/wonder-blocks-popover/src/components/__tests__/popover.test.js

```diff
@@ -97,14 +100,86 @@ describe("Popover", () => {
         );
 
         // open the popover
-        wrapper.find("[data-anchor]").simulate("click");
+        userEvent.click(screen.getByRole("button"));
 
         // Act
         // we try to close it from inside the content
-        wrapper.find("[data-close-button]").simulate("click");
+        userEvent.click(screen.getByRole("button", {name: "close popover"}));
+
+        // Assert
+        expect(screen.queryByText("Title")).not.toBeInTheDocument();
+        expect(onCloseMock).toBeCalled();
+    });
+
+    it("should close the Popover using the default close button", () => {
```

Comment:
After removing the gen snapshot tests, I noticed that we were not covering some important unit tests, so I decided to add some here.